File: tests/demo07.pl
#!/usr/bin/perl -w
# Demo for Subset4 - <>, ., .=

$string = "";
for $line (<>) {
	chomp $line;
	$line .= " hello there\n";
	print "$line";
	chomp $line;
	$string = $line."boo";
	print "STRING: $string"."hello\n";
}

File: tests/demo07.py
#!/usr/bin/python2.7 -u
import fileinput
import sys
# Demo for Subset4 - <>, ., .=

string = ""
for line in fileinput.input():
	line = line.rstrip()
	line += " hello there\n"
	sys.stdout.write("%s" % (line))
	line = line.rstrip()
	string = line+"boo"
	print "STRING: %s"  % (string) + "hello"
